scss/components/templates/_page-layout---global.scss

```diff
@@ -0,0 +1,19 @@
+.page-layout {
+  max-width: $site-max-width;
+  margin: 0 auto;
+
+  @include grid-media($grid-xs) {
+    @include grid-container($grid-xs);
+    @include grid-visual;
+  }
+
+  @include grid-media($grid-md) {
+    @include grid-container($grid-md);
+    @include grid-visual;
```

Comment:
the visual mixin is fine for preview but we will want to remove this before merging.

Reply:
OK